recipes/pelicanplatform/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "pelicanplatform" %}
+{% set version = "7.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PelicanPlatform/pelican/archive/v{{ version }}.tar.gz
+  sha256: 5a33467eaecf5e7b1338ae8107d0be75fa8156819a63a7b5cf9fc477ac724645
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+    - go-licenses
+    - nodejs >=0.18
```

Is this the right nodejs version? You probably meant 
```suggestion
    - nodejs >=18
```